quicklogic/pp3/boards.cmake

```diff
@@ -4,6 +4,7 @@ add_quicklogic_board(
   DEVICE ql-eos-s3
   PACKAGE wlcsp
   FABRIC_PACKAGE PD64
+  BIT_TO_V_EXTRA_ARGS "--device-name eos-s3 --package-name PD64"
```

Comment:
Cannot these args be inferred by other args? e.g. `FABRIC_PACKAGE` and `DEVICE`

Reply:
They can. I have fixed that.